styles.css

```diff
@@ -0,0 +1,20 @@
+body {
+    font-family: 'Courier New', Courier, monospace;
+    background-color: #c156ff;
+    margin-left: 70px;
+    margin-right: 70px;
+    margin-top: 20px;
+  }
+  
+  #header {
+    display: flex;
+    background-color: #d8ff56;
+    justify-content: space-between;
+    width: 100%;
+  }
+  #nav-bar {
```

Comment:
Remember the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Reply:
- Kindly apply my previous suggestions here as well 🤗